packages/ui/src/hooks/use-card.ts

```diff
@@ -0,0 +1,26 @@
+import { useAppearance } from '~/contexts';
```

Comment:
Can you add some comments to this file describing what the hook does, when you'd want to use it, etc? Also what do you think about calling it `useCardDecoration`? With good docs I don't really think the name matters, but it feels like these values aren't important for the actual usage of the card, so calling the hook `useCard` feels a bit bigger than what the use case actually is.

Reply:
Let me know what you think here https://github.com/clerk/javascript/pull/3984/commits/5083de9c9869a5892421a8d21815c55a75ca9404